frontend/src/components/filterable-table.js

```diff
@@ -184,15 +190,52 @@ function FilterableTable(props) {
     if (selected == null) {
         tableComponent = <ReactTable columns={columns} data={filteredData} />;
     }
+
+    function removeFilterString(filterString) {
+        const updatedFilterStrings = filterStrings.filter(
+            (oldString) => oldString !== filterString
+        );
+        setFilterStrings(updatedFilterStrings);
+    }
+
+    function addFilterString(filterString) {
+        const formattedFilterString = filterString.toLowerCase();
+        if (!filterStrings.includes(formattedFilterString)) {
+            const newFilterStrings = Array.from(filterStrings); // so we don't mutate the filterStrings object
+            newFilterStrings.push(formattedFilterString);
+            setFilterStrings(newFilterStrings);
```

Comment:
This whole thing can become `setFilterStrings([...newFilterStrings, formattedFilterString])`

Reply:
yes, it should